easybuild/easyconfigs/l/libxml2/libxml2-2.9.4-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
```

Comment:
@wpoely86 for libxml2 with Python bindings, please use the existing `libxml2` easyblock

We can enhance that easyblock so it can be used for *all* `libxml2` easyconfigs, to resolve the split-brain situation we have now where `libxml2` without Python is built via `ConfigureMake`